pyproject.toml

```diff
@@ -1,6 +1,59 @@
 [build-system]
-requires = [
-  "setuptools >= 40.9.0",
-  "wheel",
+requires = ["setuptools"]
```

Comment:
Out of curiosity, is `wheel` now automatically included? Your build has a `.whl` but I always had to add it here for `python -m build` to pick it up.